tests/modules/script_flags.c

```diff
@@ -0,0 +1,48 @@
+#include "../../src/redismodule.h"
```

Comment:
```suggestion
#include "redismodule.h"
```

Reply:
yea, this was just to get clion to work nicely temporarily

Reply:
@sjpotter , still a relevant fix...